wiki/Rules/Explicit_Content/id.md

```diff
@@ -0,0 +1,32 @@
+# Konten eksplisit
+
+![Tampilan beatmap yang mengandung konten eksplisit](img/explicit-tag.jpg "Contoh beatmap yang ditandai dengan label 'eksplisit'.")
+
+Ketika membuat beatmap, para mapper dapat menyematkan label 'eksplisit' pada beatmap-beatmap buatan mereka di situs web osu! untuk menandai bahwa beatmap-beatmap yang bersangkutan mengandung konten eksplisit.
+
+Artikel ini menjelaskan seputar berbagai hal yang terkait dengan dengan label tersebut beserta tata cara penggunaannya.
+
+## Hal-hal apa saja yang termasuk ke dalam konten eksplisit?
+
+osu! mengizinkan pemakaian **konten audio** yang bersifat eksplisit (baik dari segi *tema* ataupun *lirik* lagu) hingga batasan tertentu.
+
+**Di sisi lain, osu! tidak membenarkan penggunaan gambar dan video yang bersifat senonoh - seluruh elemen visual yang ada tetap harus tunduk pada [Pertimbangan Konten Visual](/wiki/Rules/Visual_Content_Considerations).**
+
+Pengecualian di sini hanya diberikan pada elemen-elemen visual senonoh yang mengandung unsur estetika seni (*artistic use*) dalam nuansa dan konteks pemakaiannya.
+
+Pada intinya, selama dilabeli sebagaimana mestinya, lagu-lagu yang bersifat eksplisit dapat dipergunakan dan disebarluaskan dalam bentuk beatmap di dalam lingkungan osu!.
+
+## Lirik lagu yang seperti apa yang dikategorikan eksplisit?
+
+Untuk dapat dikategorikan eksplisit, suatu lagu harus memiliki lirik yang mengandung hujatan, cacian, atau kata-kata kasar lainnya dalam porsi yang signifikan. Apabila kata-kata kasar tersebut hanya muncul sekali atau dua kali dalam keseluruhan lirik lagu yang ada, maka lagu yang bersangkutan tidak perlu untuk dilabeli eksplisit selama tema lagu yang dimaksud berada pada ranah PG-13 atau lebih rendah. Apabila kamu tidak yakin apakah suatu lagu terbilang eksplisit atau tidak, sebaiknya kamu mengasumsikan bahwa lagu tersebut adalah lagu yang eksplisit demi kemaslahatan bersama.
```

Comment:
```suggestion
Lirik lagu yang dapat dikategorikan eksplisit; memiliki lirik yang mengandung hujatan, cacian, atau kata-kata kasar lainnya dalam porsi yang signifikan. Apabila kata-kata kasar tersebut hanya muncul sekali atau dua kali dalam keseluruhan lirik lagu yang ada, maka lagu yang bersangkutan tidak perlu untuk dilabeli eksplisit selama tema lagu yang dimaksud berada pada ranah PG-13 atau lebih rendah. Apabila kamu tidak yakin apakah suatu lagu terbilang eksplisit atau tidak, sebaiknya kamu mengasumsikan bahwa lagu tersebut adalah lagu yang eksplisit demi kemaslahatan bersama.
```

Reply:
the use of `harus` seems to mean force to choose the songs that have explicit lyrics to get mapped.

Reply:
yeah that's what the "Untuk dapat dikategorikan eksplisit" words are for ;~;

`Untuk dapat dikategorikan eksplisit, suatu lagu harus memiliki lirik yang mengandung hujatan...` --> `To be categorized as explicit, the song's lyrics has to contain profanity...`